src/test/java/seedu/address/logic/parser/IVolunteerParserTest.java

```diff
@@ -50,9 +53,14 @@ public void parseCommand_clear() throws Exception {
 
     @Test
     public void parseCommand_delete() throws Exception {
-        VolunteerDeleteCommand command = (VolunteerDeleteCommand) parser.parseCommand(
+
+        Command command = parser.parseCommand(
                 VolunteerDeleteCommand.COMMAND_WORD + " " + INDEX_FIRST_VOLUNTEER.getOneBased());
-        assertEquals(new VolunteerDeleteCommand(INDEX_FIRST_VOLUNTEER), command);
+        if (command instanceof EventDeleteCommand) {
+            assertEquals(new EventDeleteCommand(INDEX_FIRST_EVENT), command);
+        } else {
+            assertEquals(new VolunteerDeleteCommand(INDEX_FIRST_VOLUNTEER), command);
+        }
```

Comment:
wouldnt this just run the else part? can try:

```suggestion
        VolunteerDeleteCommand command = (VolunteerDeleteCommand) parser.parseCommand(
                VolunteerDeleteCommand.COMMAND_WORD + " " + INDEX_FIRST_VOLUNTEER.getOneBased());
        assertEquals(new VolunteerDeleteCommand(INDEX_FIRST_VOLUNTEER), command);
        
        EventDeleteCommand command2 = (EventDeleteCommand) parser.parseCommand(
                EventDeleteCommand.COMMAND_WORD + " " + INDEX_FIRST_EVENT.getOneBased());
        assertEquals(new EventDeleteCommand(INDEX_FIRST_EVENT), command2);
```

tell me if there's any errors or issues with this